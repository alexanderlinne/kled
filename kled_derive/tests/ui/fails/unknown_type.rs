#[macro_use]
extern crate kled_derive;

#[operator(type = "Fail")]
pub struct Map<ItemOut, UnaryOp> {}

fn main() {}
