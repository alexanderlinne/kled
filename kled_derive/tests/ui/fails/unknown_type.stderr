error: Unknown literal value `Fail` at type
 --> $DIR/unknown_type.rs:4:19
  |
4 | #[operator(type = "Fail")]
  |                   ^^^^^^
